freeipa/src/main/java/com/sequenceiq/freeipa/util/CrnService.java

```diff
@@ -1,5 +1,11 @@
 package com.sequenceiq.freeipa.util;
 
+import com.sequenceiq.cloudbreak.auth.ThreadBasedUserCrnProvider;
```

Comment:
unnecessary re-ordering of imports

Reply:
why un-necessary? And moreover its due to settings from CDP and CB. I cant do much there.